wonjong/models/likeDao.js

```diff
@@ -0,0 +1,21 @@
+const appDataSource = require('./appDataSource');
+
+const createLike = async (userId, postId) => {
+  try {
+    return await appDataSource.query(
+      `INSERT INTO likes(
+                user_id,
+                post_id
+            ) VALUES (?, ?);`,
+      [userId, postId]
+    );
+  } catch (err) {
+    const error = new Error('INVALID_DATA_INPUT');
+    error.statusCode = 500;
```

Comment:
상태코드 500은 예외처리를 안 했을 때 사용하는 코드 입니다. 따라서 예외처리를 의도한 코드에서는 400번대 상태코드를 사용해 주세요.
해당 코드뿐만 아니라 전체적으로 500을 사용한 곳이 있으면 수정해 주세요.
```suggestion
    error.statusCode = 400;
```